### PR TITLE
Dynamically handle product and product version for both RHEL and Fedora systems

### PR DESCRIPTION
**Pull request by redhatrises**
This combines the rhel7-branch and master branch way of dynamically handling product and product version in anaconda. The goal with this PR is to no longer have separate code for fedora and rhel products.

### CODE REVIEW

**Comment by redhatrises** [comment]
Updated dynamic code from PR #12.

File: org_fedora_oscap/common.py [hunk 1]
```diff
@@ -54,6 +55,13 @@
 
 SSG_DIR = "/usr/share/xml/scap/ssg/content/"
 SSG_XCCDF = "ssg-rhel7-xccdf.xml"
+if product.productName.lower() != 'anaconda':
+    if product.productName.lower() == 'fedora':
+        SSG_XCCDF = "ssg-fedora-xccdf.xml"
+    elif product.productName.lower() == 'rhel':
+        SSG_XCCDF = "ssg-rhel%s-xccdf.xml" %(product.productVersion)
```
Comment (mpreisler) [inline org_fedora_oscap/common.py]: This is a useless branch, it does the same as the branch below it

Comment (redhatrises) [reply org_fedora_oscap/common.py]: @mpreisler should we just do this:

```
if product.productName.lower() != 'anaconda':
    if product.productName.lower() == 'fedora':
        SSG_XCCDF = "ssg-fedora-xccdf.xml"
    else:
        SSG_XCCDF = "ssg-%s%s-xccdf.xml" %(product.productName.lower(), product.productVersion)
```

Comment (mpreisler) [reply org_fedora_oscap/common.py]: Yeah and please make it pep8 compliant.

**Comment by redhatrises** [comment]
@mpreisler made suggested fixes.

**Comment by mpreisler** [comment]
ACK

**Pull request merged**